cmd/sqlcmd/sqlcmd.go

```diff
@@ -118,23 +120,172 @@ func (a SQLCmdArguments) authenticationMethod(hasPassword bool) string {
+
+	rootCmd.Flags().StringVarP(&args.Password, "password", "P", "", "Obsolete. The initial passwords must be set using the SQLCMDPASSWORD environment variable or entered at the password prompt.")
+
+	// Using PersistentFlags() for ErrorSeverityLevel due to data type uint8 , which is not supported in Flags()
+	rootCmd.PersistentFlags().Uint8VarP(&args.ErrorSeverityLevel, "error-severity-level", "V", 0, "Controls the severity level that is used to set the ERRORLEVEL variable on exit.")
+	// ScreenWidth is *int type , need to find corresponding Flag Type in PFlag
+	// int *defaultSomeFlag;
+	// 	rootCmd.Flags().IntVarP(args.ScreenWidth, "ScreenWidth", "w", 0, "Specifies the screen width for output. Sets the SQLCMDCOLWIDTH variable.")
+	// rootCmd.Flags().NoOptDefVal = ""
+
+	//Adding a validator for checking the enum flags
+	rootCmd.Flags().SetNormalizeFunc(func(f *pflag.FlagSet, name string) pflag.NormalizedName {
+		switch name {
+		case "application-intent":
+			value := strings.ToLower(getFlagValueByName(f, name))
+			switch value {
+			case "default", "readonly":
+				return pflag.NormalizedName(name)
+			default:
+				_, err := normalizeWithError(value, fmt.Errorf("--application-intent must be one of \"default\",\"readonly\",\"vert\",\"vertical\" but got \"%s\"", value))
+				if err != nil {
+					fmt.Fprintln(os.Stderr, err)
+				}
+				return pflag.NormalizedName("")
+			}
+		case "encrypt-connection":
+			value := strings.ToLower(getFlagValueByName(f, name))
+			switch value {
+			case "default", "false", "true", "disable":
+				return pflag.NormalizedName(name)
+			default:
+				_, err := normalizeWithError(value, fmt.Errorf("--encrypt-connection must be one of \"default\",\"false\",\"true\",\"disable\" but got \"%s\"", value))
+				if err != nil {
+					fmt.Fprintln(os.Stderr, err)
+				}
+				return pflag.NormalizedName("")
+			}
+		case "format":
+			value := strings.ToLower(getFlagValueByName(f, name))
+			switch value {
+			case "horiz", "horizontal", "vert", "vertical":
+				return pflag.NormalizedName(name)
+			default:
+				_, err := normalizeWithError(value, fmt.Errorf("--format must be one of \"horiz\",\"horizontal\",\"vert\",\"vertical\" but got \"%s\"", value))
+				if err != nil {
+					fmt.Fprintln(os.Stderr, err)
+				}
+				return pflag.NormalizedName("")
+			}
+		case "errors-to-stderr":
+			value := getFlagValueByName(f, name)
+			switch value {
+			case "-1", "0", "1":
+				return pflag.NormalizedName(name)
+			default:
+				_, err := normalizeWithError(value, fmt.Errorf("--errors-to-stderr must be one of \"-1\",\"0\",\"1\" but got \"%s\"", value))
+				if err != nil {
+					fmt.Fprintln(os.Stderr, err)
+				}
+				return pflag.NormalizedName("")
+			}
+		}
+		return pflag.NormalizedName(name)
+	})
+
+}
+
+func getFlagValueByName(flagSet *pflag.FlagSet, name string) string {
+	var value string
+	flagSet.VisitAll(func(f *pflag.Flag) {
+		if f.Name == name {
+			value = f.Value.String()
+			return
```

Comment:
>		return [](http://example.com/codeflow?start=1&length=8)

return value

Reply:
> 
This return exit from loop and then func return value.
I am not able to access http://example.com/codeflow?start=1&length=8